package-lock.json

```diff
@@ -1,6724 +0,0 @@
-{
```

Comment:
You should not delete this file, you should just remove any changes to this file from your commit.  You can checkout the version fo the file from the master branch if necessary.